.github/workflows/static.yml

```
@@ -37,6 +37,7 @@ jobs:
       - run: npm run build
         env:
           NOTION_API_KEY: ${{ secrets.NOTION_API_KEY }}
+          NOTION_DATABASE_ID: "c30f99b58b264a42b613070b56c88300"
```

```suggestion
          NOTION_DATABASE_ID: ${{ secrets.NOTION_DATABASE_ID }}
```

Let's have it as a secret.